### PR TITLE
add foreman plugin foreman_expire_hosts

### PR DESCRIPTION
**Pull request by timogoebel**
@domcleal : As discussed on https://github.com/theforeman/puppet-foreman/pull/454 . Would you mind taking a close look into this? I've never even started foreman-installer, so this is just a lucky guess.

### CODE REVIEW

**Comment by domcleal** [comment]
Could you also add a migration file like this one? https://github.com/theforeman/foreman-installer/blob/develop/config/foreman.migrations/20160405135043_foreman_proxy_discovery.rb, but the second line should use a symbol for the second hash, e.g. `scenario[:mapping][:'foreman::plugin::expire_hosts']` (it was a bug.)

**Comment by timogoebel** [comment]
@domcleal : Added the migration and finally found the documentation for this.

File: config/foreman.migrations/160614145332_foreman_expire_hosts.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+answers['foreman::plugin::expire_hosts'] ||= false
```
Comment (domcleal) [inline config/foreman.migrations/160614145332_foreman_expire_hosts.rb]: This file should be named 20160614145332... so it orders correctly with the existing files.

Comment (timogoebel) [inline config/foreman.migrations/160614145332_foreman_expire_hosts.rb]: @domcleal: Sorry for this, i just copied the command from the kafo docs. Pushed a new commit.

Is `date +%y%m%d%H%M%S` the correct command on your machine?
I need `date +%Y%m%d%H%M%S` both on OS X and RHEL...

Comment (domcleal) [inline config/foreman.migrations/160614145332_foreman_expire_hosts.rb]: %Y's needed, I didn't notice the Kafo recommendation until after I added migrations here.  I used the full year to match Rails migrations and most date formats, and think I still prefer it.

**Comment by domcleal** [comment]
Merged, thanks @timogoebel!

**Pull request merged**